File: AppGenerateFiles/helpdesk/FrontModel/ModelComponent/ViewActividadesParticipantesViewModel.js
//@ts-check
import { EntityClass } from "../../WDevCore/WModules/EntityClass.js";
import { WAjaxTools } from "../../WDevCore/WModules/WComponentsTools.js";
//@ts-ignore
import { ModelProperty } from "../../WDevCore/WModules/CommonModel.js";
class ViewActividadesParticipantes_ModelComponent extends EntityClass {
   constructor(props) {
       super(props, 'ViewHelpdesk');
       for (const prop in props) {
           this[prop] = props[prop];
       }
   }
   /**@type {ModelProperty}*/ Id_Case = { type: 'number' };
   /**@type {ModelProperty}*/ Titulo = { type: 'text' };
   /**@type {ModelProperty}*/ Descripcion = { type: 'text' };
   /**@type {ModelProperty}*/ Estado = { type: 'SELECT', Dataset: ["ACTIVO", "INACTIVO"]  };
   /**@type {ModelProperty}*/ Id_Perfil = { type: 'number' };
}
export { ViewActividadesParticipantes_ModelComponent }

File: AppGenerateFiles/helpdesk/Model/ViewActividadesParticipantesViewModel.cs
using CAPA_DATOS;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
namespace DataBaseModel {
   public class ViewActividadesParticipantes : EntityClass {
       public int? Id_Case { get; set; }
       public string? Titulo { get; set; }
       public string? Descripcion { get; set; }
       public string? Estado { get; set; }
       public int? Id_Perfil { get; set; }
   }
}
